sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -266,13 +243,13 @@ def _delay_failure(start_time: datetime.datetime) -> None:
         time.sleep((min_time - (current_time - start_time)).total_seconds())
 
 
-def _get_headers(request_type, **kwargs) -> str:
+def _get_headers(request_type, replica_count, **kwargs) -> str:
```

Comment:
I don't think there is any reason to use `kwargs` instead of keyword-only arguments here? It would be great to slowly clean up its use over time as we touch APIs.

Reply:
Updated at least headers to not use kwargs